src/data/clients.ts

```diff
@@ -303,4 +303,13 @@ export const clients: ClientData[] = [
       "https://carback.us/rick/contact/"
     ],
   },
+  {
+    description: "Mad implementation: JAM in the code-golfing stats language Jelly",
+    homepage: "https://github.com/ltfschoen/jam",
```

Comment:
Is this correct? How are you related with the effort of ltfschoen?

Reply:
Oops, no. Not correct.
String was intended to be blank (after copying and pasting).

Closing PR in any case as i won't have time to work on this..